### PR TITLE
Make es6

### PR DESCRIPTION
**Pull request by yujinjcho**
Don't know too much about ES6. Let me know if you see anything else worth changing.

### CODE REVIEW

File: webapp.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-var dataManager = require('./app_modules/dataManager');
+const express = require('express');
+const bodyParser = require('body-parser');
+const session = require('express-session');
+const pushups = require('./app_modules/pushups');
+const dataManager = require('./app_modules/dataManager');
 
-var app = express();
+const app = express();
 var db;
```
Comment (achiu8) [inline webapp.js]: Can this be `let`?

File: webapp.js [hunk 2]
```diff
@@ -49,64 +44,56 @@ app.use(passport.session());
...
 app.get('/login', passport.authenticate('facebook'));
 
-app.get('/logout', function(req, res) {
-  req.session.destroy(function(err) {
-    res.redirect('/');
-  });
-});
+app.get(
+  '/logout',
+  (req, res) => {req.session.destroy((err) => { res.redirect('/') })}
```
Comment (achiu8) [inline webapp.js]: Can remove curly braces.

File: webapp.js [hunk 3]
```diff
@@ -49,64 +44,56 @@ app.use(passport.session());
...
-  }
+  (req, res) => dataManager.handleUser(req.user, res, db)
 );
 
-MongoClient.connect(config.mongoDev, function(err, dbConnection) {
+MongoClient.connect(config.mongoDev, (err, dbConnection) => {
   db = dbConnection;
-  var server = app.listen(config.port, function() {
-    var port = server.address().port;
+  let server = app.listen(config.port, () => {
```
Comment (achiu8) [inline webapp.js]: Why can't these be `const`?

**Comment by achiu8** [comment]
@yujinjcho Are you going to convert the other files separately?

**Comment by yujinjcho** [comment]
@achiu8 yeah. Just wanted to do this one first.

**Comment by yujinjcho** [comment]
@achiu8 Ok. Made those changes. Once I push do you automatically get an email?

**Comment by yujinjcho** [comment]
@achiu8 Also, would it make more sense to make a new branch for converting the components to ES6 or just keep doing on this branch?

**Comment by achiu8** [comment]
@yujinjcho Yeah, I get an email for any changes you push / comments you leave. Can create other branches to convert other files. I'll merge this since it's good already.

**Pull request merged**